conanfile.py

```diff
@@ -13,11 +13,11 @@ class DbSQLiteAdapterConan(ConanFile):
 	exports_sources = "*", "!build*", "!*.yml", "!*.md", "!*.in", "!ci", "!.gitattributes", "!.gitignore", "!LICENSE"
 
```

Comment:
options of test_package/conanfile.py should be deleted

Reply:
Right, corrected and tested locally